iot-hub/Samples/device/PnpDeviceSamples/Thermostat/Program.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Azure.Devices.Client.Samples
 {
     public class Program
     {
-        // DTDL interface used: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json
+        // DTDL interface used: https://github.com/Azure/iot-plugandplay-models/blob/main/dtmi/com/example/temperaturecontroller-2.json
```

Comment:
@ericmitt Did you need to make this comment change? This sample still uses Thermostat model, not temperatureController model